python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -511,21 +511,25 @@ def __call__(self, fn: Callable[..., Any]) -> AssetsDefinition:
         )
 
 
-def compute_required_resource_keys(
-    required_resource_keys: Set[str],
-    resource_defs: Mapping[str, ResourceDefinition],
+def compute_required_resource_keys_for_underlying_op(
```

Comment:
Are these changes meant to be in this PR? They seem unrelated to "Move spec resolution into mapper"?

Reply:
agree. i removed these changes and restacked 👍🏻